README.md

```diff
@@ -73,12 +73,15 @@ to Maven local:
 
 - `git clone https://github.com/awslabs/smithy.git`
 - `cd smithy`
-- `./gradlew publishToMavenLocal`
+- `./gradlew clean build publishToMavenLocal`
```

Comment:
This command should stay the same.

Reply:
Reverted in https://github.com/trivikr/smithy-typescript/commit/54dc53f69bd07e0269867d816a7f4852f68ba692